### ISSUE TITLE
[Backend] Python?

### ISSUE THREAD

**Issue by CtrlC-Root** [andersevenrud/OS.js-v2#38]
I know a lot of people, myself included, that would prefer to avoid PHP and Node anywhere in our deployments if at all possible. Would anyone other than me be interested in a Python server backend?

**Comment by andersevenrud** [comment]
@CtrlC-Root I've started on the documentation of the server-side of OS.js. If there is anything you would like to see in there (like if you have any questions, that would be great) -- let me know! :smile:

http://andersevenrud.github.io/OS.js-v2/doc/server/

**Comment by CtrlC-Root** [comment]
@andersevenrud That would be really great. I took a brief look at implement a Python backend and had to put it down for now because of other commitments. It's not very clear to me 1) where you configure OS.js to use a specific backend, 2) what kind of API the backend should present. I mean, I can take educated guesses at it from the code I've looked at but it would be great if that was documented.

**Comment by CtrlC-Root** [comment]
@andersevenrud Now that documentation is up, I'll take another crack at writing a Python backend.

**Comment by andersevenrud** [comment]
@CtrlC-Root 

1) You don't configure it, you start your server of choice (see `INSTALL.md` "Setting up a server" section)

2) Now that I have documented the basic communication between Client and Server, an educated guess should do it. (The only docs missing is the VFS functions actually).


But I feel the current documentation is not dense enough and a bit unclear (It was so friggin late when I wrote that). So if you have *any questions at all, just send them my way* so I can improve the docs.

The best way to get an understanding of how this works is simply to use the Network tools in the browser. Just filter the "XHR" requests and look at the request/response headers :)
